### PR TITLE
Force render with just that camera on CreateScreenshot

### PR DESCRIPTION
**Pull request by carolhmj**
If a camera is specified as an argument, fall back on the CreateScreenshotUsingRenderTarget function.
Related forum issue: https://forum.babylonjs.com/t/screenshot-tool-does-not-seem-to-respect-camera-argument/31167
Related Playground: https://playground.babylonjs.com/#M6HFGB

### CODE REVIEW

File: packages/dev/core/src/Misc/tools.ts [hunk 1]
```diff
@@ -772,6 +772,35 @@ export class Tools {
         );
     }
 
+    static DownloadBlob(blob: Blob, fileName?: string) {
```
[the remark this answers is not in the document]
Comment (sebavan) [reply packages/dev/core/src/Misc/tools.ts:775]: LGTM otherwise

[the remark this answers is not in the document]
Comment (RaananW) [reply packages/dev/core/src/Misc/tools.ts:775]: > NIT: missing comments :-)

Still on it. Sorry for the delay here

[the remark this answers is not in the document]
Comment (RaananW) [reply packages/dev/core/src/Misc/tools.ts:781]: Feel free to resolve this comment when you are ready to merge

[the remark this answers is not in the document]
Comment (sebavan) [reply packages/dev/core/src/Misc/tools.ts:781]: @RaananW weird nothing in the build failed for this one ???

**Review by sebavan — APPROVED** [review]

File: packages/dev/core/src/Misc/tools.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (RaananW) [reply packages/dev/core/src/Misc/tools.ts:781]: as written, still on it. a bit delayed on that.

[the remark this answers is not in the document]
Comment (sebavan) [reply packages/dev/core/src/Misc/tools.ts:781]: ohhhhhhhhh my bad I thought you meant you were adding the comments, not the tools :-) got it

**Pull request merged**